tests/conftest.py

```diff
@@ -63,6 +63,15 @@ def init_base_world():
     BaseWorld.apply_config('payloads', BaseWorld.strip_yml(os.path.join(CONFIG_DIR, 'payloads.yml'))[0])
 
 
+@pytest.fixture(scope='session')
+def celery_config():
+    db_path = os.getcwd() + '/data/celery.db'
```

Comment:
up to implementer but personal preference is f-strings or os.path.join() for these path definitions